templates/service.yaml

```
@@ -37,7 +37,7 @@ spec:
   healthCheckNodePort: {{ $.Values.service.healthCheckNodePort }}
   {{- end }}
   {{- if $.Values.service.ipFamily }}
-  ipFamily: {{ $.Values.service.ipFamily }}
+  ipFamilyPolicy: {{ $.Values.service.ipFamily }}
```

Wouldn't it make sense to also rename the field in the `values.yaml` file?
The following would also remain downward compatibility:

```suggestion
  ipFamilyPolicy: {{ or $.Values.service.ipFamilyPolicy $.Values.service.ipFamily }}
```